.github/workflows/phan.yml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Before adding a new too in CI, we must first have the tool working from command line.

Can you make instad oa PR to add files (README) into dev/tools/phan to explain how to install and run phan from command line ?

Reply:
SCRIPT addeed, README.md added.